src/util/placeTypes.ts

```diff
@@ -122,22 +122,30 @@ export type Services =
 export type PlaceWithAccesibilityData = Place & {
```

Comment:
can we just remove this now?

Reply:
If it isn't being used anywhere else, then yet. Otherwise, that will be taken care of in #292 . Will let you know if I need to do it here @carpenterd777

Reply:
Just scanned the repo and this type is being used all over and is therefore out of scope for me to remove references to it here. @carpenterd777